content/en/docs/reference/access-authn-authz/service-accounts-admin.md

```diff
@@ -89,8 +95,10 @@ The ServiceAccount admission controller will add the following projected volume
 
 This projected volume consists of three sources:
 
-1. A ServiceAccountToken acquired from kube-apiserver via TokenRequest API. It will expire after 1 hour by default or when the pod is deleted. It is bound to the pod and has kube-apiserver as the audience.
-1. A ConfigMap containing a CA bundle used for verifying connections to the kube-apiserver. This feature depends on the `RootCAConfigMap` feature gate, which publishes a "kube-root-ca.crt" ConfigMap to every namespace. `RootCAConfigMap` feature gate is graduated to GA in 1.21 and default to true. (This flag will be removed from --feature-gate arg in 1.22)
+1. A ServiceAccountToken acquired from kube-apiserver via TokenRequest API. It will expire
+   after 1 hour by default or when the pod is deleted. It is bound to the pod and has
+   `kube-apiserver` as the audience.
+1. A ConfigMap containing a CA bundle used for verifying connections to the kube-apiserver.
 1. A DownwardAPI that references the namespace of the pod.
```

Comment:
**Aside**: we could rewrite this. “A ConfigMap” is clearly a single instance of a countable thing; “a DownwardAPI” isn't really, because:
- you can't find these as objects in the Kubernetes API
- if we keep it, we're mixing volume data sources (eg config map data source) with the place that data source looks for data (the ConfigMap, for example).